doc/nrf/libraries/others/supl_os_client.rst

```diff
@@ -120,8 +120,8 @@ API callback functions
 
 Callback functions are needed to achieve platform independence.
 If your application uses the SUPL client library, you must implement these callback functions.
-You can find an implementation example in the :ref:`gps_with_supl_support_sample` sample.
-The implementation is located in the :file:`samples\\nrf9160\\gps\\src\\` files.
+For an example of the implementation, see the :ref:`gnss_sample` sample.
+The implementation is located in the :file:`samples\nrf9160\gnss\src\` directory.
```

Comment:
```suggestion
The implementation is located in the :file:`samples/nrf9160/gnss/src/` directory.
```
Maybe we need slashes instead of backslashes. Looks like we have them that way in other docs. Not sure this caused the build fail, but could well be this.